src/utils/combineReducers.js

```diff
@@ -97,9 +98,16 @@ export default function combineReducers(reducers) {
     }
   });
 
+  if (sanityError) {
+    console.error(sanityError.message);
+  }
+
```

Comment:
I've played with this a bit more using devtools and I don't think the `console.error` is helpful. Throwing `sanityError` logs the stack trace from where the error was created in `combineReducers`, so the two stack traces in the log are actually the same.